Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -165,6 +175,26 @@ void didReceiveAuthenticationChallenge(Ref<WebKit::AuthenticationChallengeProxy>
 
         [m_delegate.getAutoreleased() didReceiveAuthenticationChallenge:nsURLChallenge completionHandler:completionHandler.get()];
     }
+#if ENABLE(SCREEN_TIME)
+    void getScreenTimeURLs(const String& identifer, CompletionHandler<void(HashSet<URL>&&)>&& callback) const final
+    {
+        if (![PAL::getSTWebHistoryClass() instancesRespondToSelector:@selector(fetchAllHistoryWithCompletionHandler:)])
+            return;
+
+        STWebHistoryProfileIdentifier const WebHistoryProfileIdentifier = identifer;
```

Comment:
I don't think `const` is needed. also should be `webHistoryProfileIdentifier` since it's a variable name.